### PR TITLE
ci: DEV-2123: Add frontend command

### PR DESCRIPTION
**Pull request by nikitabelonogov**

### CODE REVIEW

File: .github/workflows/frontend-command.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - name: Checkout on chat command
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GIT_PAT }}
+          repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.client_payload.pull_request.head.ref }}
+          submodules: 'recursive'
```
Comment (farioas) [inline .github/workflows/frontend-command.yml:22]: Remove

File: .github/workflows/frontend-command.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "${{ env.NODE }}"
```
Comment (farioas) [inline .github/workflows/frontend-command.yml:28]: Add cache

File: .github/workflows/frontend-command.yml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          git config --global user.name '${{ github.event.client_payload.github.actor }}'
+          git config --global user.email '${{ github.event.client_payload.github.actor }}@users.noreply.github.com'
+          
+          git add -A
+          git status -s
+          if git diff-index --quiet HEAD; then
+            echo "::set-output name=changes::no"
+            exit 0
+          else 
+            git commit -m "[frontend] Get build"
```
Comment (farioas) [inline .github/workflows/frontend-command.yml:64]: Rich linkgs to commits

Comment (farioas) [reply .github/workflows/frontend-command.yml:64]: Add module name in commit msg, eg Get build lsf

**Review by farioas — CHANGES_REQUESTED** [review]

File: .github/workflows/frontend-command.yml (hunk 1 above)
Comment (nikitabelonogov) [reply .github/workflows/frontend-command.yml]: done

File: .github/workflows/frontend-command.yml (hunk 2 above)
Comment (nikitabelonogov) [reply .github/workflows/frontend-command.yml:27]: done

File: .github/workflows/frontend-command.yml (hunk 3 above)
Comment (nikitabelonogov) [reply .github/workflows/frontend-command.yml]: done

**Review by farioas — APPROVED** [review]

**Pull request merged**